client/coral-embed-stream/src/index.js

```diff
@@ -12,6 +12,10 @@ import localStore, {injectReducers} from 'coral-framework/services/store';
 import AppRouter from './AppRouter';
 import {pym} from 'coral-framework';
 
+import {loadTranslations} from 'coral-framework/services/i18n';
+
+loadTranslations();
```

Comment:
Why do we need to explicitly `loadTranslations`? All translations are already in the bundle.